Watchman.Discord/Areas/Responses/Services/ResponsesGetterService.cs

```diff
@@ -25,14 +29,39 @@ public IEnumerable<Response> GetResponsesFromBase()
 
         public IEnumerable<Response> GetResponsesFromResources()
         {
+            Log.Information("Initializing default responses...");
+            var managerMethods = typeof(ResponsesManager).GetMethods();
             var defaultResponses = typeof(Devscord.DiscordFramework.Framework.Commands.Responses.Resources.Responses).GetProperties()
                 .Where(x => x.PropertyType.Name == "String")
                 .Select(prop =>
                 {
                     var onEvent = prop.Name;
                     var message = prop.GetValue(prop)?.ToString();
-                    return new Response(onEvent, message, DEFAULT_SERVER_ID);
+                    var parameters = managerMethods.First(info => info.Name == onEvent).GetParameters().Skip(1);
+                    var availableVariables = new List<string>();
+                    foreach (var parameter in parameters)
+                    {
+                        switch (parameter.ParameterType.Name)
+                        {
+                            case "Contexts":
+                                availableVariables.Add("user");
+                                availableVariables.Add("channel");
+                                availableVariables.Add("server");
+                                break;
+                            case "UserContext":
+                                availableVariables.Add("user");
+                                break;
+                            case "DiscordServerContext":
+                                availableVariables.Add("server");
+                                break;
+                            default:
+                                availableVariables.Add(parameter.Name);
+                                break;
+                        }
+                    }
+                    return new Response(onEvent, message, DEFAULT_SERVER_ID, availableVariables.ToArray());
                 });
+            Log.Information("Default responses initialized.");
```

Comment:
jeszcze w ramach inicjalizacji przemyśl aktualizacje obecnych responses

hmm 
możliwe że osobna kolekcja w bazie będzie miała tutaj sens
albo wrzucić to na sztywno do wszystkich `Where(x => x.OnEvent == )`

osobna kolekcja to dodatkowa operacja na bazie
na rozmiar bazy jakoś nie narzekamy
a inicjalizacja odpala się rzadko